tests/src/integration/common/wskdeploy.go

```diff
@@ -47,17 +47,17 @@ func (wskdeploy *Wskdeploy) RunCommand(s ...string) ([]byte, error) {
 }
 
 func (wskdeploy *Wskdeploy) Deploy(manifestPath string, deploymentPath string) ([]byte, error) {
-	return wskdeploy.RunCommand("-m", manifestPath, "-d", deploymentPath)
+	return wskdeploy.RunCommand("-i ", "false", "-m", manifestPath, "-d", deploymentPath)
```

Comment:
@lzbj I think the intent here was to disable interactive mode, but now like Matt is saying it is disabled by default, so you don't need this flag "-i" here at all. If you specify "-i" now on wskdeploy CMD line, interactive mode is turned on.